algorithms/linfa-pls/benches/pls.rs

```diff
@@ -0,0 +1,33 @@
+use criterion::{criterion_group, criterion_main, BenchmarkId, Criterion};
+use linfa::traits::Fit;
+use linfa::Dataset;
+use linfa_datasets::generate::make_dataset;
+use linfa_pls::PlsRegression;
+use statrs::distribution::{DiscreteUniform, Laplace};
+
+#[allow(unused_must_use)]
+fn perform_pls(num_rows: usize) {
+    let feat_distr = Laplace::new(0.5, 5.).unwrap();
+    let target_distr = DiscreteUniform::new(0, 5).unwrap();
+    let num_feats: usize = 5;
+    let dataset = make_dataset(num_rows, num_feats, feat_distr, target_distr);
+    let targets = dataset.targets.into_shape((num_rows, 1)).unwrap();
```

Comment:
Maybe we should include multi-target datasets in the benchmarks.

Reply:
I can see if it is a seamless addition